File: ProgrammingFundamentals/C#/04.MethodsDebuggingAndTroubleshootingCode/Lab/P06_CalculateTriangleArea/P06_CalculateTriangleArea.cs
using System;

namespace P06_CalculateTriangleArea
{
    class P06_CalculateTriangleArea
    {
        static void Main(string[] args)
        {
            double width = double.Parse(Console.ReadLine());
            double height = double.Parse(Console.ReadLine());
            Console.WriteLine(GetTriangleArea(width, height));
        }

        static double GetTriangleArea(double width, double height)
        {
            double result = width * height / 2;
            return result;
        }
    }
}

File: ProgrammingFundamentals/Java/04.MethodsDebuggingTroubleshootingCode/Lab/src/P06_CalculateTriangleArea.java
import java.text.DecimalFormat;
import java.util.Scanner;
public class P06_CalculateTriangleArea {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("#.#######");
        double width = Double.parseDouble(console.nextLine());
        double height = Double.parseDouble(console.nextLine());
        System.out.println(df.format(getTriangleArea(width, height)));
    }

    static double getTriangleArea(double width, double height) {
        double result = width * height / 2;
        return  result;
    }
}
